applications/nrf5340_audio/src/audio/streamctrl_broadcast_source.c

```diff
@@ -321,7 +321,7 @@ void streamctrl_send(void const *const data, size_t size, uint8_t num_ch)
 	int ret;
 	static int prev_ret;
 
-	struct encoded_audio enc_audio = {.data = data, .size = size, .num_ch = num_ch};
+	struct le_audio_encoded_audio enc_audio = {.data = data, .size = size, .num_ch = num_ch};
```

Comment:
This name I find too elaborate. We could send this data over WiFi.

Reply:
That is true. But this struct is defined in le_audio.h, and it will only be used for LE Audio.
And as I understand it, it is a general naming scheme in le_audio.h to start with "le_audio"